### PR TITLE
Added series key to filter by annotate values

### PR DESCRIPTION
**Pull request by ar4s**

### CODE REVIEW

File: src/ralph/dashboards/models.py [hunk 1]
```diff
@@ -39,6 +39,15 @@ class Graph(NamedMixin, TimeStampMixin, models.Model):
     params = JSONField(blank=True)
     active = models.BooleanField(default=True)
 
+    def pop_annotate_filters(self, filters):
+        annotate_filters = {}
+        annotate_keys = [
+            f for f in filters.keys() if f.startswith('series')
+        ]
+        for key in annotate_keys:
+            annotate_filters.update({key: filters.pop(key)})
```
Comment (ghost) [inline src/ralph/dashboards/models.py]: If you wish :) you can merge those two loops to one ;)

Comment (mkurek) [inline src/ralph/dashboards/models.py]: +1

**Comment by szok** [comment]
:ok:

**Comment by mkurek** [comment]
Any description about this functionality? How should this work? What would be the effect?

**Comment by ghost** [comment]
:ok:

**Pull request merged**